docs/source/applications.rst

```diff
@@ -36,13 +42,33 @@ add_file_to_db
    :members:
 
 
+add_value_from_json_to_db
+==========================
+
+.. automodule:: add_value_from_json_to_db
+   :members:
+
+
 compare_cumulative_psf
```

Comment:
<img width="818" alt="Screenshot 2024-04-29 at 16 31 06" src="https://github.com/gammasim/simtools/assets/50831535/e06d2d2c-d14a-420f-92bf-5bec784049e8">
There is a very large empty space in the example code of this application and apparently in all applications as well. the snippet code is likely not recognizing a new line. Worth a look

Reply:
Well spotted, forgot that one has to apply `\\` for line breaks in docstrings. Fixed this.